07-javascript.js

```diff
@@ -67,6 +67,20 @@ let str4 = "briefly"
 let str5 = "Offensive Word"
 
 // START
+String.prototype.bowlderize = function() {
+  let newString = "";
+  for(let i = 0; i < this.length; i++){
+    let letter = this.charAt(i);
+    if(letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u'
+    || letter == 'A' || letter == 'E' || letter == 'I' || letter == 'O' || letter == 'U'){
+      newString += "*";
+    }
+    else{
+      newString += letter;
+    }
+  }
+  return newString;
+}
```

Comment:
I'd like to help you develop a kind of spidey sense, where you look at code you've written and think "hmmm... it works, but it seems excessively long or complicated. There must be a better way. In this case, maybe

~~~ js
String.prototype.bowlderize = function() {
  return this.replace(/[aeiou]/ig, "*")
}
~~~

The rubric says there's no deduction, as the tests pass, but I want you to keep your eyes open for things like this going forward.

---